pkg/dependency/parser/java/pom/pom.go

```diff
@@ -268,6 +268,8 @@ func (d pomDependency) Resolve(props map[string]string, depManagement, rootDepMa
 		}
 		if len(dep.Exclusions.Exclusion) == 0 {
 			dep.Exclusions = managed.Exclusions
+		} else if len(managed.Exclusions.Exclusion) > 0 {
+			dep.Exclusions.Exclusion = append(dep.Exclusions.Exclusion, managed.Exclusions.Exclusion...)
```

Comment:
It also looks like we should always be merging exceptions.
```suggestion
		// `mvn` always merges exceptions for pom and parent
		dep.Exclusions.Exclusion = append(dep.Exclusions.Exclusion, managed.Exclusions.Exclusion...)
```